release/models/system/openconfig-system-grpc-tunnel.yang

```diff
@@ -0,0 +1,348 @@
+module openconfig-system-grpc-tunnel {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/system/grpc-tunnel";
+  prefix "oc-sys-grpc-tun";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system { prefix oc-sys; }
+  import openconfig-network-instance { prefix oc-netinst; }
+  import openconfig-inet-types { prefix oc-inet; }
+
+  organization
+    "OpenConfig working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module adds configuration and operational state relating to gRPC
+    tunnel (https://github.com/openconfig/grpctunnel/) to invoke one or
+    more dialout TCP over gRPC tunnels to bypass situations in which
+    direct dialin gRPC communications are not feasible.";
+
```

Comment:
(nit) extra blank line.

Reply:
Addressed in commit https://github.com/openconfig/public/pull/666/commits/fae9ac3b3a62b798acba53272400175c2e927690